packages/gravity/kube-testing/src/plan/transport-spec.ts

```diff
@@ -2,7 +2,8 @@
 // Copyright 2023 DXOS.org
```

Comment:
why `-spec.ts` suffix? should this be `.test.ts`?

Reply:
Should we rename this pacakge?

Reply:
it's not `.test.ts` because it's not a test runnable by `mocha`, but rather a spec for a test that is executed via our own testing framework (we use it within kube-testing to test signal, transport & echo)